plugins/Morpheus/stylesheets/less/_variables.less

```diff
@@ -17,6 +17,8 @@
 @theme-color-background-highContrast:  #202020;
 @theme-color-base-series:              #ee3024;
 
+@theme-color-border: @color-silver-l80;
```

Comment:
theme-color-border is not really expressive. Border of what? I reckon we have many different border columns. I think it would be better to use `@color-silver-l80` directly instead of `@theme-color-border`. If not, at least we should move it to a `_colors.less` and we shouldn't make it API.